src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,189 @@
+		empty folder: <gold>%s<reset> nie ma w sobie żadnych włączonych skryptów.
+	enable:
+		all:
+			enabling: Włączanie wszystkich wyłączonych skryptów...
+			enabled: Pomyślnie włączono & zinterpretowano wszystkie wyłączone skrypty.
+			error: Znaleziono %s bł¦ąd¦ędów¦ podczas interpretacji wyłączonych skryptów!
+			io error: Nie udało się wczytać żadnych skryptów (niektóe skrypty mogły mieć zmienioną nazwę i zostaną one włączone po restarcie serwera): %s
+		single:
+			already enabled: <gold>%s<reset> jest już włączony! Użyj <gray>/<gold>skript <cyan>reload <red>%s<reset> aby odświeżyć skrypt, jeśli był zmieniany.
+			enabling: Włączanie <gold>%s<reset>...
+			enabled: Pomyślnie włączono & zinterpretowano <gold>%s<reset>.
+			error: Znaleziono %2$s bł¦ąd¦ędów¦ podczas interpretacji <gold>%1$s<red>!
+			io error: Nie udało się włączyć <gold>%s<red>: %s
+		folder:
+			empty: <gold>%s<reset> nie zawiera w sobie żadnych wyłączonych skryptów.
+			enabling: Włączanie <gold>%2$s <reset>skrypt¦¦ów¦ in <gold>%1$s<reset>...
+			enabled: Pomyślnie włączono & zinterpretowano <gold>%2$s<reset> poprzednio wyłączonych skryptów <gold>%1$s<reset>.
+			error: Znaleziono %2$s bł¦ąd¦ędów¦ podczas interpretacji skryptów w folderze <gold>%1$s<red>!
+			io error: Błąd podczas włączania skryptów w folderze <gold>%s<red> (niektóre skrypty mogą zostać włączone po restarcie serwera): %s
+	disable:
+		all:
+			disabled: Pomyślnie wyłączono wszystkie skrypty.
+			io error: Nie udało się nazwać wszystkich skryptów ponownie - niektóre skrypty mogą zostać włączone po restarcie serwera: %s
+		single:
+			already disabled: <gold>%s<reset> jest już wyłączony!
+			disabled: Pomyślnie wyłączono <gold>%s<reset>.
+			io error: Nie udało się nazwać <gold>%s<red> ponownie, skrypt może zostać włączony po restarcie serwera: %s
+		folder:
+			empty: <gold>%s<reset> nie zawiera żadnych włączonych skryptów.
+			disabled: Pomyślnie wyłączono <gold>%2$s<reset> skrypt¦¦ów¦ w <gold>%1$s<reset>.
+			io error: Nie udało się wyłączyć wszystkich skryptów w <gold>%s<red> (niektóre skrypty mogą zostać wyłączone po restarcie serwera): %s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> aktualizacj¦a¦i¦ zostało wydanych od wydania tej wersji (<gold>%s<r>) Skript'a:
+			# 	footer: Aby pokazać listę zmian wersji, napisz <gold>/skript update changes <version><reset>
+			# invalid version: Nie ma listy zmian dla wersji <gold>%s<red>
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>strona %s. z %s. Wpisz <gold>/skript update changes %s<gray> by przewinąć stronę (podpowiedź: użyj strzałki w górę na klawiaturze)
+	info:
+		aliases: Aliasy Skript'a mogą zostać znalezione tutaj: <aqua>https://github.com/SkriptLang/skript-aliases
+		documentation: Dokumentacja Skript'a może zostać znaleziona tutaj: <aqua>https://skriptlang.github.io/Skript
+		version: Wersja Skript: <aqua>%s
```

Comment:
```suggestion
		version: Wersja Skript'a: <aqua>%s
```
Jak już wszędzie odmieniamy to tu też warto (wstawiam z apostrofem dla konsekwencji z tym co teraz mamy)